.circleci/config.yml

```diff
@@ -127,31 +124,50 @@ jobs:
       - run: helm package -d .cr-release-packages/ --version ${CIRCLE_TAG:1} pachyderm/etc/helm/pachyderm # cr package did not work with embedded chart
       - run: cr upload -o pachyderm -r helmchart --skip-existing
       - run: cd helmchart && cr index -o pachyderm -r helmchart -c https://helm.pachyderm.com --package-path ../.cr-release-packages --push
-  hub:
-    resource_class: large
+  load_test:
+    resource_class: xlarge
     machine:
       image: ubuntu-2004:202101-01
+    environment:
+      GOOGLE_PROJECT_ID: build-release-001
```

Comment:
Does `bq` just automatically pick these variables up?

Reply:
yea, the circle ci gcp cli orb does when it sets up the cli on agent

Reply:
got it, so that orb is doing like `gcloud init` or whatever.